httpbin doesn't build with aeson 2

```
Preprocessing executable 'httpbin' for wreq-0.5.3.3..
Building executable 'httpbin' for wreq-0.5.3.3..
[1 of 2] Compiling HttpBin.Server   ( httpbin/HttpBin/Server.hs, dist/build/httpbin/httpbin-tmp/HttpBin/Server.dyn_o )

httpbin/HttpBin/Server.hs:74:57: error:
    • Couldn't match type ‘Data.Text.Internal.Text’
                     with ‘Data.Aeson.Key.Key’
      Expected: [aeson-2.0.3.0:Data.Aeson.Types.Internal.Pair]
        Actual: [(Data.Text.Internal.Text, Value)]
    • In the first argument of ‘object’, namely ‘cs’
      In the expression: object cs
      In the expression: ("cookies", object cs)
   |
74 |   respond $ \obj -> return $ obj <> [("cookies", object cs)]
   |                                                         ^^
```

--- a/httpbin/HttpBin/Server.hs
+++ b/httpbin/HttpBin/Server.hs
@@ -9,6 +9,7 @@ import Control.Applicative ((<$>))
 import Control.Monad.IO.Class (liftIO)
 import Data.Aeson (Value(..), eitherDecode, object, toJSON)
 import Data.Aeson.Encode.Pretty (Config(..), Indent(Spaces), defConfig, encodePretty')
+import Data.Aeson.Key (fromText)
 import Data.ByteString.Char8 (pack)
 import Data.CaseInsensitive (original)
 import Data.Maybe (catMaybes, fromMaybe)
@@ -69,7 +70,7 @@ setCookies = do
 
 listCookies = do
   cks <- rqCookies <$> getRequest
-  let cs = [(decodeUtf8 (cookieName c),
+  let cs = [(fromText(decodeUtf8 (cookieName c)),
              toJSON (decodeUtf8 (cookieValue c))) | c <- cks]
   respond $ \obj -> return $ obj <> [("cookies", object cs)]
 

--- a/tests/UnitTests.hs
+++ b/tests/UnitTests.hs
@@ -5,7 +5,6 @@
 
 module UnitTests (testWith) where
 
-import Control.Arrow (first)
 import Control.Applicative ((<$>))
 import Control.Concurrent (forkIO, killThread)
 import Control.Concurrent.MVar (newEmptyMVar, putMVar, takeMVar)
@@ -34,6 +33,7 @@ import Test.Framework (Test, defaultMain, testGroup)
 import Test.Framework.Providers.HUnit (testCase)
 import Test.HUnit (assertBool, assertEqual, assertFailure)
 import qualified Control.Exception as E
+import qualified Data.Aeson.KeyMap as KM
 import qualified Data.CaseInsensitive as CI
 import qualified Data.HashMap.Strict as HMap
 import qualified Data.Text as T
@@ -86,8 +86,8 @@ cikey :: AsValue t => T.Text -> Traversal' t Value
 cikey i = _Object . toInsensitive . ix (CI.mk i)
   where
     toInsensitive = iso toCi fromCi
-    toCi = HMap.fromList . map (first CI.mk) . HMap.toList
-    fromCi = HMap.fromList . map (first CI.original) . HMap.toList
+    toCi = HMap.mapKeys CI.mk . KM.toHashMapText
+    fromCi = KM.fromHashMapText . HMap.mapKeys CI.original
 
 basicGet Verb{..} site = do
   r <- get (site "/get")

--- a/wreq.cabal
+++ b/wreq.cabal
@@ -139,7 +139,7 @@ executable httpbin
     buildable: False
   else
     build-depends:
-      aeson >= 0.7,
+      aeson >= 2.0,
       aeson-pretty >= 0.8.0,
       base >= 4.5 && < 5,
       base64-bytestring,

